chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -374,7 +375,10 @@ In any [exponential family](https://en.wikipedia.org/wiki/Exponential_family) mo
 This fact makes computing gradients easy in practice.
 -->
 
-*dịch đoạn phía trên*
+Nói cách khác, gradient chính là hiệu giữa xác xuất dự đoán đúng của mô hình, được ký hiệu là $P(y \mid x)$, và lớp thật sự đúng của dữ liệu, được ký hiệu là $y$.
+Ta sẽ thấy tương tự như trong hồi quy, gradient cũng chính là hiệu giữa kết quả của dữ liệu thu thập được $y$ và kết quả được ước lượng $\hat{y}$. Đây hoàn toàn không phải là sự trùng hợp.
+Trong bất kỳ mô hình nào [Mô hình họ mũ](https://en.wikipedia.org/wiki/Exponential_family), gradient của hàm log-likelihood đều được định nghĩa một cách chính xác.
```

Comment:
`exponential family` chắc mình phải hỏi thêm
```suggestion
Trong mọi [mô hình họ lũy thừa](https://en.wikipedia.org/wiki/Exponential_family), gradient của hàm log-likelihood đều có dạng như thế này.
```

Reply:
chỗ này mình vẫn nghĩ nên dịch là họ mũ do `Exponential` đó giờ vẫn dịch là mũ (giống kiểu Exponential distribution là phân phối mũ) mà dịch họ mũ không thì không ai hiểu nên mình thêm chữ Mô hình vô :))

Reply:
yeah thực ra mình cũng không rành về toán lắm, từ này chắc để mấy chuyên gia giải quyết vậy